dockstore-webservice/src/main/java/io/dockstore/webservice/core/RSSToolPath.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 OICR
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dockstore.webservice.core;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class is only used to get data from the database in a more type-safe way
+ * @author gluu
+ * @since 1.8.0
+ */
+public class RSSToolPath extends RSSEntryPath {
```

Comment:
At first blush, this looks identical to RSSWorkflowPath and very similar to RSSEntryPath and they're both treated the same way. 
Can they be merged?

Reply:
Tool has registry, namespace, and toolname
Workflow has sourceControl (SourceControl enum), organization, and workflowname

I can rename these fields but it might be confusing if someone wanted to get the organization of the workflow and whatever I rename it to might not be obvious.

Reply:
Yeah, I'm either or. 

I have a ticket out to look at standardizing on the workflow names, its a bit weird but it simplifies a lot of code